rpcserver.go

```diff
@@ -1337,6 +1337,7 @@ func (r *rpcServer) GetTransactions(context.Context,
 			Amount:           tx.Value.ToBTC(),
```

Comment:
Hmm, yeah I think I prefer using satoshis just about everywhere in the RPC interface uniformly.